pkg/runtime/wfengine/workflow.go

```diff
@@ -454,12 +466,23 @@ func (wf *workflowActor) runWorkflow(ctx context.Context, actorID string, remind
 		}
 		return newRecoverableError(fmt.Errorf("failed to schedule a workflow execution: %w", err))
 	}
```

Comment:
if a workflow fails to be scheduled, shouldn't the metric be recorded as execution failed? or do we have another metrics for schedule failed?

Reply:
The metric being added here is for execution. For scheduling, another metric will be added later on: https://github.com/dapr/dapr/issues/7109#issuecomment-1868047411